plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -254,16 +254,22 @@ export default () => {
     [backendURL, backendAccessToken],
   );
 
-  const handleStoryShare = useCallback((story: Story) => {
+  const handleStoryShare = useCallback(() => {
+    setCurrentPage("share");
+  }, []);
```

Comment:
This isn't really handling story sharing, so can you rename this to handleGoToShare or something?